containers/proxy-squid-image/uyuni-configure.py

```diff
@@ -2,17 +2,73 @@
 
 import os
 import re
+import shutil
+
 import yaml
 
+
+def set_cache_parameters(content, squid_config):
+    # max_cache_size_mb
+    content = re.sub(r"cache_dir aufs .*",
+                     f"cache_dir aufs /var/cache/squid {str(squid_config['max_cache_size_mb'])} 16 256",
```

Comment:
We need to be able to read the old way of setting that value in the config file for compatibility reason.